Mlem/Views/Shared/Translation Sheet.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  Translation Sheet.swift
+//  Mlem
+//
+//  Created by tht7 on 22/06/2023.
+//
+
+import Foundation
+import SwiftUI
+
+struct TranslationSheet: View {
+    @Binding var textToTranslate: String?
+    @Binding var shouldShow: Bool
+
+    var body: some View {
+        ZStack(alignment: .topTrailing) {
+            InAppWebView(url: URL(string: "https://translate.google.com/?tl=\(Locale.current.language.languageCode?.identifier ?? "")&q=\(textToTranslate ?? "Missing Text")")!)
```

Comment:
Force unwrapping a constructed string URL init is generally not a good idea.

Reply:
While I generally agree with you, here we can guarantee the link is valid 100% of the times, because we control the prefix and the prefix is valid

If it bothers you in terms of codes style, I'll add the condition